torch/_functorch/functional_call.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+from typing import Dict, Union, Any, Tuple
+
+import torch
+import torch.nn as nn
+from torch import Tensor
+
+
+def functional_call(
+    module: 'torch.nn.Module',
+    parameter_and_buffer_dicts: Union[Dict[str, Tensor], Tuple[Dict[str, Tensor]]],
+    args: Union[Any, Tuple],
+    kwargs: Dict[str, Any] = None,
+):
+    parameters_and_buffers = parameter_and_buffer_dicts if isinstance(parameter_and_buffer_dicts, dict) else {}
+    if isinstance(parameter_and_buffer_dicts, tuple):
```

Comment:
Should we assert that parameter_and_buffer_dicts is a tuple of two things here? Or is it fine for it to have arbitrary number of dicts? (at which point we should rename it `member_dicts` or something)

Reply:
Yeah this should be able to take in an arbitrary number of dictionaries (I haven't written it out, but I thought it might help integration with tensordict). I'll change the signature and name to reflect this